src/functions/baseUrl.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (req, res) => {
+    console.log(req)
+    return hostname = req.headers.host;
```

Comment:
Are we using this function, else we can remove it.

Reply:
@rjvim made the changes as per your comments, please review it again.